Makefile

```diff
@@ -63,6 +63,13 @@ conftest-test: ## Run unit tests with conftest
 fmt: ## Apply default formatting to all rego files. Use before you commit
 	@opa fmt . --write
 
+amend-fmt: fmt ## Apply default formatting to all rego files then amend the current commit
+	@git diff $$(git ls-files '*.rego')
+	@echo "Amend commit '$$(git log -n1 --oneline)' with the above diff?"
+	@read -p "Hit enter to continue, Ctrl-C to abort."
+	git add $$(git ls-files '*.rego')
```

Comment:
`-u` might be useful here to avoid accidentally adding untracked files.

Reply:
I think it's fine since we're doing the `git ls-files` instead of `find .`

Reply:
You are correct